src/components/SettingsDrawer.tsx

```diff
@@ -55,6 +61,13 @@ const SettingsDrawer: FC<{ open: boolean; onClose: () => void }> = ({
     (state) => state.appPreferences.etherDecimalPlaces
   );
 
+  const currentDisplayedTestNets = useAppSelector((state) =>
+    sortBy(
```

Comment:
This `sortBy` might run many times. Please create a memoized selector: https://redux-toolkit.js.org/api/createSelector